### ISSUE TITLE
Thumbnails for e-rara and e-manuscripta missing

### ISSUE THREAD

**Issue by klokan** [klokantech/kartenportal-tickets#120]
A new problem was detected today: All images except of the Ryhiner images have gone, i.e. e-rara and e-manuscripta images are no longer shown.
See 
http://search.kartenportal.ch/#id=045311757
http://search.kartenportal.ch/#id=131358952

**Comment by klokan** [comment]
It seems the links to the thumbnails were completely removed from SwissBib and from the MARC records.
OAI-PMH update which runs every night automatically removed these from Kartenportal as well.

**Our implementation is expecting the link to thumbnail in the MARC 956$u and the link must end with ".jpg".**

The mentioned example record in SwissBib does not contain the link to thumbnail in the MARC at all:
http://www.swissbib.ch/TouchPoint/perma.do?q=0%3D%22045311757%22

This has been already implemented, running fine, validated and accepted in #30. Now there is a change on the side of SwissBib. If they restore the links in MARC records as they were before the images will appear again on Kartenportal too.

If we should work on this again and AGAIN despite the problem is not on our side I need the definitive, official and final rule for getting the direct link to thumbnail image from SwissBib from @zb-markus which we should implement.

Because the links to thumbnails were completely removed from SwissBib MARC, I don't see a way how to restore these. From my point of view this ticket can be closed now, because there is nothing what KlokanTech can do.

**Comment by zb-markus** [comment]
I am sorry I missed the fact that Swissbib has removed the 956 fields we need. If I had noted this before, I would not have opened this issue. When I accepted #30, everything was fine at the time. I did simply not expect that Swissbib makes such dramatic changes and removes the 956 fields completely. Look, it is sometimes very difficult to see whether a certain problem is a regression problem within Geosearch or not. You have correctly pointed out the cause. We will therefore talk to Swissbib on that issue.
